sticker-graph/sticker_graph/conv_model.py

```diff
@@ -1,9 +1,8 @@
+from enum import Enum
```

Comment:
Minor: blank line between standard library and other imports

Reply:
I am fine with that, but is there a standard? `autopep8` does not enforce a blank line.

Reply:
Fixed, thanks!

Reply:
Afaik, there are various standards. Tensorflow seems to do:

```
future

standard lib

external

local
```

Reply:
I like it! That's my Rust ordering (except for future).